### PR TITLE
allow resource dropdown to select storageclass in dropdown

### PR DESCRIPTION
**Pull request by weirdwiz**

### CODE REVIEW

File: packages/shared/dropdown/ResourceDropdown.tsx [hunk 1]
```diff
@@ -155,9 +155,10 @@ const ResourceDropdown: ResourceDropdown = <T extends unknown>({
 
   const onClick = React.useCallback(
     (event: React.SyntheticEvent<HTMLDivElement>) => {
-      const resourceUID = event?.currentTarget?.id;
+      // not a good way to select resource, but we need the id to be resourcename-link for CI tests
```
[the remark this answers is not in the document]
Comment (weirdwiz) [reply packages/shared/dropdown/ResourceDropdown.tsx]: pushed an update

**Comment by GowthamShanmugam** [comment]
Tested, fix is working fine

image

**Review by GowthamShanmugam — APPROVED** [review]

**Pull request merged**